.travis.yml

```diff
@@ -1,14 +1,18 @@
-sudo: false
-language: java
-dist: trusty
-jdk: oraclejdk8
-cache:
-  directories:
-  - "$HOME/.m2"
-install: true
-script:
-- mvn clean verify
-- sed -i'' -e "s/\$VERSION/${TRAVIS_TAG:=0.0.0}/" descriptor.json
-- sed -i'' -e "s/\$DATE/$(date +%Y-%m-%d)/" descriptor.json
-notifications:
-  webhooks: https://fathomless-fjord-24024.herokuapp.com/notify
+dist: bionic
+
+language: java 
+jdk:
+  - oraclejdk11
+  - openjdk10

```

Comment:
As discussed: Please remove openjdk10 because it fails (and seems deprecated here).

Reply:
fixed https://github.com/tthuem/VariantSync/pull/53/commits/cbfe2a3cf2b3b665254c6ab6d6b802179d5c2fde